### ISSUE TITLE
Support for Laravel Nova 4.x

### ISSUE THREAD

**Issue by chrillep** [kraenkvisuell/nova-astrotranslatable#5]

**Comment by chrillep** [comment]
@manogi do you have any plans for this? or sync upstream from https://github.com/outl1ne/nova-translatable which has support?

**Comment by bastihilger** [comment]
Hey - sorry for replying so late. At them moment I have zero time for my Nova packages, and I think this will stay this way until the end of the year. Please feel free to fork, if you like.

**Comment by chrillep** [comment]
> 

yeah i hear u. 
thanks for the reply.
i'll fork and rebase upstream.

**Comment by chrillep** [comment]
ok so my fork supports nova 4. but needs more testing in order for me to be comfortable releasing it :) 


https://github.com/yaytrade/nova-astrotranslatable

**Comment by majdcodes** [comment]
@chrillep any update?

**Comment by chrillep** [comment]
Nope. Not working on this anymore. And won't in the future neither 😅
